parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -68,27 +82,23 @@ public void setMinMaxFromBytes(byte[] minBytes, byte[] maxBytes) {
   }
 
   @Override
-  public boolean isSmallerThan(long size) {
-    return !hasNonNullValue() || ((min.length() + max.length()) < size);
+  String toString(Binary value) {
+    // TODO: have separate toString for different logical types?
+    return value.toStringUsingUTF8();
```

Comment:
Delegating to type would be better here, and would work when this Binary is used for Decimal.